### DISCUSSION TITLE
can i resume training changing to another model to get best of 2 world?

### DISCUSSION THREAD

**Discussion by DimensionY** [TheLastBen/fast-stable-diffusion#437] [Q&A]
Hello,
example of what i mean
i'm training a subject with  runwayml/stable-diffusion-v1-5 for 1000 steps  at 30%
then i change it to  nitrosocke/redshift-diffusion and i resume it for another 1000 steps at 40%

is posible? Will the training understand that I change to another style so that he learns to do the subject in that other style too?

ty

**Comment by TheLastBen** [comment]
hi,
if you resume a training it won't use the first model anymore, it will use the produced model in the folder "/content/models/(session_name)

**Reply by DimensionY** [reply]
oh ok ty
